.github/workflows/scripts/ti_build/vulkan.py

```diff
@@ -12,23 +12,35 @@
 
 
 # -- code --
-@banner("Setup Vulkan 1.3.236.0")
+@banner("Setup Vulkan 1.4.309.0")
 def setup_vulkan():
     u = platform.uname()
     if u.system == "Linux":
-        url = "https://sdk.lunarg.com/sdk/download/1.3.236.0/linux/vulkansdk-linux-x86_64-1.3.236.0.tar.gz"
-        prefix = get_cache_home() / "vulkan-1.3.236.0"
+        url = "https://sdk.lunarg.com/sdk/download/1.4.309.0/linux/vulkansdk-linux-x86_64-1.4.309.0.tar.xz"
+        prefix = get_cache_home() / "vulkan-1.4.309.0"
         download_dep(url, prefix, strip=1)
         sdk = prefix / "x86_64"
         os.environ["VULKAN_SDK"] = str(sdk)
         path_prepend("PATH", sdk / "bin")
         path_prepend("LD_LIBRARY_PATH", sdk / "lib")
         os.environ["VK_LAYER_PATH"] = str(sdk / "etc" / "vulkan" / "explicit_layer.d")
+    elif (u.system, u.machine) in (("Linux", "arm64"), ("Linux", "aarch64")):
```

Comment:
from the title of this PR, this PR is only for updating existing deps, not migrating to add ARM?

Reply:
No, it is upgrading everything to generate arm wheels. https://pypi.jetson-ai-lab.dev/jp6/cu128/taichi/1.8.0